lib/msal-browser/src/app/Configuration.ts

```diff
@@ -96,7 +99,9 @@ const DEFAULT_LOGGER_OPTIONS: LoggerOptions = {
 const DEFAULT_SYSTEM_OPTIONS: BrowserSystemOptions = {
     loggerOptions: DEFAULT_LOGGER_OPTIONS,
     networkClient: BrowserUtils.getBrowserNetworkClient(),
-    windowHashTimeout: DEFAULT_POPUP_TIMEOUT_MS
+    windowHashTimeout: DEFAULT_POPUP_TIMEOUT_MS,
+    iframeHashTimeout: DEFAULT_IFRAME_TIMEOUT_MS,
+    loadFrameTimeout: 0
```

Comment:
Could we dynamically set this to `500` if we are in IE/Edge?

Reply:
done.